### PR TITLE
cypress artifacts & node versions

### PR DESCRIPTION
**Pull request by mitchelvanbever**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -53,10 +53,31 @@ jobs:
         run: yarn test:ci
   cypress-run:
     runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        node: [10, 12, 14]
```
Comment (mpeyper) [inline .github/workflows/test.yml:58]: I'm not sure what benefit we get out of running the cypress tests on multiple versions of node as the tests execute in a browser environment?

**Comment by mitchelvanbever** [comment]
Not sure if we need to change some stuff around to accomodate for the matrix runs and the different artifacts that come from that. I think currently only one of those runs is included as artifact.

File: .github/workflows/test.yml (hunk 1 above)
Comment (mitchelvanbever) [reply .github/workflows/test.yml:58]: This was a request of @JacobMGEvans [see](https://discord.com/channels/808364903822917662/808364903822917665/815360342124265522)

Comment (mpeyper) [reply .github/workflows/test.yml:58]: it doesn't hurt so happy to take as-is.

Perhaps there is more benefit to the martrix for the `jest` tests as they execute in the the node environment? (Can be seperate PR)

**Pull request merged**